src/components/App.js

```diff
@@ -1,77 +1,85 @@
-import { ChakraProvider } from '@chakra-ui/react';
-import { Container } from '@chakra-ui/react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { useState } from "react";
+import { ChakraProvider } from "@chakra-ui/react";
+import { Container } from "@chakra-ui/react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { CartContext } from "../util/context";
+import Checkout from "./Checkout/Checkout";
```

Comment:
Kind of irrelevant, but double quotes?

Reply:
i dont know ,prettier does that automatically on save

Reply:
<img width="651" alt="image" src="https://user-images.githubusercontent.com/54900075/141414828-d8910e4e-8c84-4784-a885-24bb7145a6b0.png">

make sure single quotes are enabled in your prettier settings